tests/fiaas_skipper/deploy/test_deploy.py

```diff
@@ -101,6 +101,13 @@ def test_deploys_to_all_namespaces(self, cluster, release_channel_factory):
         deployer.deploy()
         assert 2 == deployer._deploy.call_count
 
+    def test_deploys_with_bootstrap_when_force_flag_set(self, cluster, release_channel_factory):
+        bootstrap = mock.MagicMock(name="bootstrap")
+        deployer = Deployer(cluster, release_channel_factory, bootstrap, deploy_interval=0)
+        deployer._deploy = mock.MagicMock(name="_deploy")
+        deployer.deploy(namespaces=["test1"], force_bootstrap=True)
+        assert 1 == bootstrap.call_count
```

Comment:
Matter of preference, but I like to use the `assert_called...` methods if possible instead of asserting on the call count.

Reply:
This requires a bit of refactoring of the rest of the tests as well to make them used assert_called_with. I can have a look at it in a separate PR.